pkgpanda/build/__init__.py

```diff
@@ -48,6 +56,12 @@ def run(self, cmd):
         docker.append(self.container)
         docker += cmd
         check_call(docker)
+        DockerCmd.clean(name)
```

Comment:
Here you're passing the original / base name, not the new name with random, which is leading to:
https://teamcity.mesosphere.io/viewLog.html?buildId=395451&buildTypeId=DcosIo_Dcos_BuildPr&tab=buildLog#_focus=635